app/components/mobile-dialog.hbs

```diff
@@ -0,0 +1,8 @@
+{{#if this.toShow}}
+  <dialog class="appDialog">
+      Open RDS in app
+    <br />
+    <button id="cancleBt" type="button" {{on 'click' (fn this.closeDialog)}} >Cancle</button>
+    <button id="okayBt" type="button" {{on 'click' (fn this.openRDSApp)}}>Okay</button>
```

Comment:
Please don't use abbre. for naming id and classes

Reply:
Resolve please have a look.